src/test/java/ch/uzh/soprafs22/groupmatcher/service/EmailServiceTest.java

```diff
@@ -89,7 +134,38 @@ void sendResponseVerificationEmailTest() {
         emailService.sendResponseVerificationEmail(testStudent);
         verify(mailSender).send(messageCaptor.capture());
         MimeMessage sentEmail = messageCaptor.getValue();
-        assertNotNull(sentEmail.getAllRecipients());
         assertEquals(List.of(testStudent.getEmail()), Arrays.stream(sentEmail.getAllRecipients()).map(Address::toString).toList());
     }
+
+    @SneakyThrows
+    @Test
+    void emailSubjectGroupInfoTest() {
```

Comment:
You can just combine this test (`emailSubjectGroupInfoTest`) with the test above `sendGroupInfoTest` since they test the same function (= unit test)